tests/testpreferences.cpp

```diff
@@ -159,7 +159,6 @@ void TestPreferences::testPreferences()
 
 	TEST(location->timeThreshold(), 30);
 	TEST(location->distanceThreshold(), 40);
->>>>>>> core: activate qPrefLanguage
```

Comment:
hehe. there it is again. I'd appreciate if you could refactor the commits to not add this in the first place as it will break compilation in the commits in between which could be annoying when bisecting.